src/components/calcite-pagination/calcite-pagination.tsx

```diff
@@ -190,21 +190,21 @@ export class CalcitePagination {
     );
   }
 
-  renderLeftEllipsis(iconScale: this["scale"]): VNode {
+  renderLeftEllipsis(): VNode {
     if (this.total / this.num > maxPagesDisplayed && this.showLeftEllipsis()) {
       return (
         <span class={`${CSS.ellipsis} ${CSS.ellipsisStart}`}>
-          <calcite-icon icon="ellipsis" scale={iconScale} />
+          ...
```

Comment:
Should this use the html entity? `&hellip;`

Reply:
Thanks! updated this.